### ISSUE TITLE
NTR: Modifier: Monoarticular: joint involvement (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7970]
New term request Modifier: Monoarticular: joint involvement (tirosh_130603125846): One joint; pauciarticular, monoarthritis; monoarthropathy  UMLS: 0427254

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000111
- label: Monoarticular
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Refers to a phenotypic abnormality that affects a single joint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7970
- synonyms:
